src/main/java/lotto/domain/Ticket.java

```diff
@@ -0,0 +1,24 @@
+package lotto.domain;
+
+public class Ticket {
+
+    private static final String TICKET_ERROR_MESSAGE = "최소 구매 금액은 1,000원입니다.";
+    private static final int TICKET_PRICE = 1000;
+
+    private int buyCount;
```

Comment:
```suggestion
    private final int buyCount;
```
생성뒤에 값이 변하지 않는 다는 것을 명시하고 불변을 위해 `final`로 선언해보는 것은 어떨가요?